CHANGELOG.md

```diff
@@ -6,6 +6,34 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+
+- **CUMULUS-3861**
+  - Updated React 17 to React 18
+  - Updated Enzyme tests to React Testing Library
+  - Removed test script references to Enzyme and replaced them with React compliant testing components
+    - Five Unit test scripts were converted covering 34 assertions
+      - test/components/executions/execution-events_correctStepName.js
+      - test/components/executions/execution-events_showEventHistory.js
+      - test/components/executions/execution-status.js
+      - test/components/granules/bulk-granule-modal.js
+      - test/components/header/condition.js
+
+### Changed
+
+- **CUMULUS-3870**
+  - Updated React and dependencies
+  - react 17.0.2 ⇒ 18.3.1
+  - react-ace 9.5.0 ⇒ 9.5.0
```

Comment:
react-ace is updated to ^13.0.0
We only need to call out the major version updates, the lines below can be removed.

Reply:
5d3df842